modules/calcite/src/main/codegen/config.fmpp

```diff
@@ -0,0 +1,637 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
```

Comment:
this is mainly was copy-pasted from calcite-babel module, hence it may not be paid too much attention